src/test/groovy/net/rubygrapefruit/platform/file/BasicFileEventFunctionsTest.groovy

```diff
@@ -624,4 +627,33 @@ class BasicFileEventFunctionsTest extends AbstractFileEventFunctionsTest {
         "parent of watched directory"       | { it.parentFile }
         "grand-parent of watched directory" | { it.parentFile.parentFile }
     }
+
+    def "can set log level"() {
+        given:
+        def nativeLogger = Logger.getLogger(NativeLogger.name)
+        def originalLevel = nativeLogger.level
+
+        when:
+        logging.clear()
+        nativeLogger.level = Level.FINEST
+        // Wait for the change to be picked up
+        Thread.sleep(1500)
```

Comment:
Can you extract the sleep to a method? So you don't need to duplicate the comment.